pipeline/extract.py

```diff
@@ -11,12 +11,12 @@
 """
```

Comment:
Can you add a docstring?

Reply:
There's a docstring at the top of the file, you have to expand it to see but it's there! <3